app/assets/javascripts/utils/course.js

```diff
@@ -91,6 +91,24 @@ document.onreadystatechange = () => {
     });
   }
 
+  const y = document.getElementsByClassName('delete-course-prerequisite')[0];
```

Comment:
Please use descriptive variable names rather than generic ones like `y` and `z`.

Reply:
> WikiEduDashboard/app/assets/javascripts/utils/course.js   ---   Line: 80
```
  // for use on campaign/programs page
  const x = document.querySelectorAll('.remove-course');
  for (let i = 0; i < x.length; i += 1) {
    x[i]?.addEventListener('click', (e) => {
        const confirmed = window.confirm(I18n.t('campaign.confirm_course_removal', {
          title: e.target.dataset.title,
          campaign_title: e.target.dataset.campaignTitle
        }));
        if (!confirmed) {
          e.preventDefault();
        }
    });
  }
```

Since variable x was already in use for a similar function within the file, I went ahead and used variables y and z to maintain uniformity for new functions. I'll update the y and z variables for better clearity. should i do the same for x?

Reply:
I have changed the names for all three variables - 
x -> removeCourseBtn
y -> deleteCoursePrerequisiteBtn
z ->  deleteCourseBtn